src/components/map/ConnectedGeolocation.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import { useGeolocation } from 'react-use'
+
+import { geolocationChange } from '../../redux/mapSlice'
+
+export const ConnectedGeolocation = () => {
```

Comment:
Is there a naming convention/surface-level distinction for utility components (i.e. components that provide state-related services, but don't render on the page) like this?

Reply:
Currently there isn't. I think this is a pretty rare case. I had to make this a component so that the webpage can defer requesting access to location by conditionally rendering ConnectedGeolocation.